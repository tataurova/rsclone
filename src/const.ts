export const RouteName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};
export const PageName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};

export const AuthorizationStatus = {
  AUTH: 'AUTH',
  NO_AUTH: 'NO_AUTH',
};

export const Error = {
  UNAUTHORIZED: 401,
  BAD_REQUEST: 400,
  NOT_FOUND: 404,
  SERVICE_UNAVAILABLE: 503,
  TIMEOUT_ERROR: 'timeout_error',
};

export const headCells = [
  { id: 'status', label: 'Статус' },
  { id: 'city', label: 'Город' },
  { id: 'name', label: 'ФИО' },
  { id: 'age', label: 'Возраст' },
  { id: 'date', label: 'Дата' },
  { id: 'coordinator', label: 'Координатор' },
  { id: 'people', label: 'Людей на поиске' },
  { id: 'actions', label: 'Действия', disableSorting: true },
];

export const getStatusCollection = [
  'Резерв',
  'Поиск на месте',
  'Стоп. Проверка информации',
  'Архив',
];

export const SearcherStatus = {
  way: 'В пути',
  place: 'На месте',
  home: 'Дома',
};

export const TransportStatus = {
  car: 'На машине',
  own: 'Своим ходом',
  need: 'Нужна машина',
};

export const MAX_TIMEOUT = 5000;
export const SHOW_ERROR_TIMEOUT = 3000;
