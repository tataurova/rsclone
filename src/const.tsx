export const PageName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};
