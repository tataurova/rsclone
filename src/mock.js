const data = [
  {
    id: 1,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 2,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Пертов Петр Петрович',
    city: 'Москва',
    birth: '04.09.1985',
    age: '30',
    missing: '01.10.2017',
    sign: 'Был с собакиой',
    close: 'Шорты, белоя майка',
    with: 'черная сумка',
  },
  {
    id: 3,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Кузмин Картой Петрович',
    city: 'Судлино',
    birth: '30.12.1960',
    age: '60',
    missing: '05.12.2018',
    sign: 'Хромает на леаую ногу',
    close: 'Шуба, валенки',
    with: 'трость',
  },
  {
    id: 4,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Щаруго Кирилл Федорович',
    city: 'Пирам',
    birth: '14.07.2000',
    age: '20',
    missing: '05.10.2010',
    sign: 'Заикается',
    close: 'Кепка синяя, белая майка',
    with: 'короткая стрижка',
  },
  {
    id: 5,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 6,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '25.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 7,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    sign: 'Шрам на левой щеке',
    missing: '05.12.2020',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 8,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.04.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 9,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
];

export default data;
