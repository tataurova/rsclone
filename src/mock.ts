export const data = [
  {
    id: 1,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 2,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Пертов Петр Петрович',
    city: 'Москва',
    birth: '04.09.1985',
    age: '30',
    missing: '01.10.2020',
    sign: 'Был с собакиой',
    close: 'Шорты, белая майка',
    with: 'черная сумка',
  },
  {
    id: 3,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Кузмин Картой Петрович',
    city: 'Судлино',
    birth: '30.12.1960',
    age: '60',
    missing: '05.12.2020',
    sign: 'Хромает на левую ногу',
    close: 'Шуба, валенки',
    with: 'трость',
  },
  {
    id: 4,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Щаруго Кирилл Федорович',
    city: 'Пирам',
    birth: '14.07.2000',
    age: '20',
    missing: '05.10.2020',
    sign: 'Заикается',
    close: 'Кепка синяя, белая майка',
    with: 'короткая стрижка',
  },
  {
    id: 5,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2020',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 6,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '25.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 7,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    sign: 'Шрам на левой щеке',
    missing: '05.12.2020',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 8,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.04.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 9,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },

];

export const mockActiveSearches = [
  {
    id: 1,
    status: 'Поиск на месте',
    city: 'Санкт-Петербург',
    name: 'Петр',
    age: '30',
    date: '12-01-2020',
    coordinator: 'Чиж',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }, {
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  }, {
    id: 3,
    status: 'Стоп. Проверка информации',
    city: 'Минск',
    name: 'Аркадий',
    age: '100',
    date: '01-01-2021',
    coordinator: 'Чип',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
  {
    id: 4,
    status: 'Поиск на месте',
    city: 'Москва',
    name: 'Владимир',
    age: '3000',
    date: '12-06-2020',
    coordinator: 'Дейл',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
  {
    id: 5,
    status: 'Резерв',
    city: 'Санкт-Петербург',
    name: 'Петр',
    age: '30',
    date: '12-01-2020',
    coordinator: 'Чи',
    people: [],
  },
  {
    id: 6,
    status: 'Поиск на месте',
    city: 'Киев',
    name: 'Анна',
    age: '123',
    date: '12-01-2020',
    coordinator: 'Кат',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
];
export const mockClosedSearches = [{
  id: 2,
  status: 'Архив',
  city: 'Колпино',
  name: 'Фекла',
  age: '200',
  date: '10-01-2020',
  coordinator: 'Чиж',
  people: [{
    name: 'Саша',
    status: 'way',
    transport: 'car',
    places: '1',
    time: '12:00',
    district: 'Василеостровский',
  }, {
    name: 'Катя',
    status: 'place',
    transport: 'own',
    places: '0',
    time: '19:00',
    district: 'Василеостровский',
  }, {
    name: 'Рома',
    status: 'place',
    transport: 'need',
    places: '0',
    time: '2:00',
    district: 'Калининский',
  }, {
    name: 'Илья',
    status: 'way',
    transport: 'own',
    places: '0',
    time: '15:00',
    district: 'Выборгский',
  }],
}];
