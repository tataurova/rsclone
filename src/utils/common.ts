export const extend = (a, b) => ({ ...a, ...b });

export const helperForTranslate = (value) => {
    switch (value) {
        case 'Резерв':
            return 'Reserve';
        case 'Поиск на месте':
            return 'Local search';
        case 'Стоп. Проверка информации':
            return 'Stop. Information verification';
        case 'Архив':
            return 'Archive';
    }
}

export const mockFunction = () => {
    // do nothing
};
