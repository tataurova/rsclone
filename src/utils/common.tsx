export const extend = (a, b) => ({ ...a, ...b });
