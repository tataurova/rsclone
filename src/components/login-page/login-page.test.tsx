import * as React from 'react';
import * as renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import LoginPage from './login-page';
import { mockActiveSearches, mockClosedSearches } from '../../mock';
import { AuthorizationStatus } from '../../const';
import { mockFunction } from '../../utils/common';

jest.mock('react-i18next', () => ({
  useTranslation: () => ({ t: (key) => key }),
}));

describe('<LoginPage />', () => {
  const mockStore = configureStore([]);
  const store = mockStore({
    DATA: {
      gonePeople: [],
      lookingRelativesPeople: [],
      activeSearches: mockActiveSearches,
      closedSearches: mockClosedSearches,
      isFetching: false,
      error: false,
    },
    APP: {
      page: '2',
    },
    AUTH: {
      authorizationStatus: AuthorizationStatus.NO_AUTH,
      user: '',
    },
  });

  const state = {
    loginValid: true,
    passwordValid: true,
  };
  const loginRef = React.createRef<HTMLInputElement>();
  const passwordRef = React.createRef<HTMLInputElement>();

  it('Render LoginPage', () => {
    const tree = renderer
      .create(
                <Provider store={store}>
                    <LoginPage
                        state={state}
                        onSubmit={mockFunction}
                        onChange={mockFunction}
                        loginRef={loginRef}
                        passwordRef={passwordRef}
                    />
                </Provider>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
