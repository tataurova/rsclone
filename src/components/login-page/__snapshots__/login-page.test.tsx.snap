// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> Render LoginPage 1`] = `
<main
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container"
>
  <div
    className="MuiGrid-root makeStyles-image-2 MuiGrid-item MuiGrid-grid-sm-4 MuiGrid-grid-md-7"
  />
  <div
    className="MuiPaper-root MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-5 MuiPaper-elevation6"
  >
    <div
      className="makeStyles-paper-3"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circle makeStyles-avatar-4 MuiAvatar-colorDefault"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M18 8h-1V6c0-2.76-2.24-5-5-5S7 3.24 7 6v2H6c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V10c0-1.1-.9-2-2-2zM9 6c0-1.66 1.34-3 3-3s3 1.34 3 3v2H9V6zm9 14H6V10h12v10zm-6-3c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
          />
        </svg>
      </div>
      <h1
        className="MuiTypography-root MuiTypography-h5"
      >
        Login
      </h1>
      <form
        className="makeStyles-form-5"
        onSubmit={[Function]}
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
            data-shrink={false}
            htmlFor="email"
            id="email-label"
          >
            Email
            <span
              aria-hidden={true}
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoComplete="email"
              autoFocus={true}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              id="email"
              name="email"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={true}
              type="text"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-8 MuiOutlinedInput-notchedOutline"
            >
              <legend
                className="PrivateNotchedOutline-legendLabelled-10"
              >
                <span>
                  Email
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal MuiFormControl-fullWidth"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
            data-shrink={false}
            htmlFor="password"
            id="password-label"
          >
            Password
            <span
              aria-hidden={true}
              className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoComplete="current-password"
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              id="password"
              name="password"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={true}
              type="password"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-8 MuiOutlinedInput-notchedOutline"
            >
              <legend
                className="PrivateNotchedOutline-legendLabelled-10"
              >
                <span>
                  Password
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-submit-6 MuiButton-containedPrimary MuiButton-fullWidth"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="submit"
        >
          <span
            className="MuiButton-label"
          >
            Login
          </span>
        </button>
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-body2 MuiTypography-colorPrimary"
              href="/tataurova-JS2020Q3/rsclone/signup"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              No account? Register
            </a>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-body2 MuiTypography-colorPrimary"
              href="/tataurova-JS2020Q3/rsclone/"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              Go to the main page
            </a>
          </div>
        </div>
      </form>
    </div>
  </div>
</main>
`;
